packages/controllers/src/snaps/SnapController.ts

```diff
@@ -49,6 +49,23 @@ export const controllerName = 'SnapController';
 export const SNAP_PREFIX = 'wallet_snap_';
 export const SNAP_PREFIX_REGEX = new RegExp(`^${SNAP_PREFIX}`, 'u');
 
+// APIs exposed by default to the Snap without needing permissions
+export const DEFAULT_EXPOSED_APIS = [
+  'atob',
+  'btoa',
+  'TextEncoder',
+  'TextEncoderStream',
+  'TextDecoder',
+  'TextDecoderStream',
```

Comment:
The streams don't appear to be available in Node:
```suggestion
  'TextDecoder',
```